src/Components/Categories/CategoryItem.jsx

```diff
@@ -13,7 +13,6 @@ const CategoryItem = () => {
     try {
       const categoriesData = await axios.get("/api/categories");
       dispatch({ type: "CATEGORIES", payload: categoriesData.data.categories });
```

Comment:
we should check api response status first. Only if 200 or 201 we should dispatch

Reply:
resolved here:
https://github.com/saadahmadhere/elmart/pull/6/commits/ff043b959f8db9127dca85ed3441ea33693dcab8